docs/src/main/asciidoc/firestore.adoc

```diff
@@ -0,0 +1,60 @@
+== Cloud Firestore
+
+Spring Cloud GCP provides convenience starter which automatically configures authentication settings and client objects needed to begin using the https://cloud.google.com/firestore/[Google Cloud Firestore].
+
+See https://cloud.google.com/firestore/docs/[documentation] to learn more about Cloud Firestore.
+
+=== Dependency Setup
```

Comment:
I don't see the use of a separate section for "Dependency Setup" in other docs. We usually just make it part of the introductory section.
```suggestion
```